torch/_torch_docs.py

```diff
@@ -9711,80 +9740,109 @@ def merge_dicts(*dicts):
+    correction (int): difference between the sample size and sample degrees of freedom.
+                      Defaults to `Bessel's correction`_, ``correction=1``.
     {keepdim}
     {out}
 
-.. function:: var(input, unbiased) -> Tensor
-   :noindex:
-
-Calculates the variance of all elements in the :attr:`input` tensor.
-
-If :attr:`unbiased` is ``True``, Bessel's correction will be used.
-Otherwise, the sample deviation is calculated, without any correction.
+Example:
 
-Args:
-    {input}
-    unbiased (bool): whether to use Bessel's correction (:math:`\delta N = 1`).
+    >>> a = torch.tensor(
+    ...     [[ 0.2035,  1.2959,  1.8101, -0.4644],
+    ...      [ 1.5027, -0.3270,  0.5905,  0.6538],
+    ...      [-1.5745,  1.3330, -0.5596, -0.6548],
+    ...      [ 0.1264, -0.5080,  1.6420,  0.1992]])
+    >>> torch.var(a, dim=1, keepdim=True)
+    tensor([[1.0631],
+            [0.5590],
+            [1.4893],
+            [0.8258]])
 
-Example::
+.. _Bessel's correction: https://en.wikipedia.org/wiki/Bessel%27s_correction
 
-    >>> a = torch.tensor([[-0.8166, -1.3802, -0.3560]])
-    >>> torch.var(a, unbiased=False)
-    tensor(0.1754)
 """.format(**multi_dim_common))
 
 add_docstr(torch.var_mean,
            r"""
-var_mean(input, dim, unbiased, keepdim=False, *, out=None) -> (Tensor, Tensor)
+var_mean(input, dim=None, *, correction=1, keepdim=False, out=None) -> (Tensor, Tensor)
+
+Calculates the variance and mean over the dimensions specified by :attr:`dim`.
+:attr:`dim` can be a single dimension, list of dimensions, or ``None`` to
+reduce over all dimensions.
 
-If :attr:`unbiased` is ``True``, Bessel's correction will be used to calculate
-the variance. Otherwise, the sample variance is calculated, without any
-correction.
+The variance (:math:`\sigma^2`) is calculated as
+
+.. math:: \sigma^2 = \frac{1}{N - \delta N}\sum_{i=0}^{N-1}(x_i-\bar{x})^2
+
+where :math:`x` is the sample set of elements, :math:`\bar{x}` is the
+sample mean, :math:`N` is the number of samples and :math:`\delta N` is
+the :attr:`correction`.
+""" + r"""
+
+Note:
+   In PyTorch 1.8 and earlier :func:`var_mean` accepted an :attr:`unbiased`
+   argument which was a less general form of the :attr:`correction` parameter.
+   If :attr:`unbiased` was ``True`` it implied ``correction=1`` (this was the
+   default). If :attr:`unbiased` was ``False`` then it implied
+   ``correction=0``.
+
+{keepdim_details}
 
 Args:
     {input}
     {dim}
 
 Keyword args:
-    unbiased (bool): whether to use Bessel's correction (:math:`\delta N = 1`).
+    correction (int): difference between the sample size and sample degrees of freedom.
+                      Defaults to `Bessel's correction`_, ``correction=1``.
     {keepdim}
     {out}
 
 Returns:
-    A tuple (var, mean) containing the variance and mean.
-
-.. function:: var_mean(input, unbiased) -> (Tensor, Tensor)
-   :noindex:
+    A tuple (var, mean) containing the variance and mean. Here 2
```

Comment:
I think the "Here 2" is extraneous?

Reply:
Thanks, that was left over from testing that the docs were rendering properly. Removed.